tests/board-detection.spec.js

```diff
@@ -0,0 +1,55 @@
+var test = require('tape');
+var proxyquire = require('proxyquire');
+
+test('pid board detection', function(t) {
+  t.plan(2);
+  var Avrgirl = proxyquire('../avrgirl-arduino', {
```

Comment:
to be safer from false failure reporting on Travis, this proxyquire call should also have `noCallThru` at the beginning of the chain. See an example here: https://github.com/noopkat/avrgirl-ispmkii/blob/master/tests/avrgirl-ispmkii.spec.js#L9